app/src/main/assets/about.html

```diff
@@ -1,12 +1,24 @@
 <p>
     MusicBrainz is an open music encyclopedia that collects music metadata and makes it available to the public.
+    <br><br>
+    It is the ultimate source of music information by allowing anyone to contribute and releasing the data under open licenses.
 </p>
 <h5>What does it do?</h5>
 <p>
-    This app gives you easy access to MusicBrainz data on your Android device. Scan a barcode to look up release information or search for artists and releases. Registered users can submit tags, ratings, barcodes and manage their collections. Every user can tag audio files.
+    MusicBrainz for Android is a mobile app for viewing MusicBrainz data on the Android platform. It allows you to:
+
+    <li> View release information by scanning a barcode</li>
+    <li> Search for information about artists, releases, release groups,labels, recordings, instruments, and events.</li>
+    <li> View collections</li>
+    <li> Tag audio files like Picard</li>
+    <li> Send Releases to your Picard</li>
+    <li> Donate to the MetaBrainz Foundation via PayPal</li>
 </p>
 <h5>Development</h5>
-<p>MusicBrainz for Android was first created in 2010-11 as a part of GSoC by Jamie McDonald. The project went stagnant for a few years.<br> In 2019, development on the project started again and was later taken up as another GSoC project by amCap1712.</br> <br> In GSoC 2020,combined effort of SomalRudra & amCap1712 revamped the app design, improved overall infrastructure & has been successful to implement the key feature:Tagger </br> <br> The source code is available on GitHub. Bug reports and comments are welcomed, just tap 'Feedback' in the menu.</br>
+<p> The source code for this app is available on GitHub at https://github.com/metabrainz/musicbrainz-android<br><br>
+    A tutorial on the usage of Picard functionalities on the app can be found at https://picard-docs.musicbrainz.org/en/tutorials/android_app.html<br><br>
```

Comment:
The tutorial line can probably be moved out of the Development section. Maybe add a Tutorials section just before development and put this there.

Reply:
Done